### ISSUE TITLE
I can't connect to MAL or to Hummingbird

### ISSUE THREAD

**Issue by bjk14** [tofuness/Toshocat#14]
Always says "Incorrect credentials", but I have checked my password multiple times.

**Comment by adifaidz** [comment]
Mine too. I think its because the api url has now changed, like how hummingbird changed to kitsu. For MAL, I can't figure out the reason. Realized this one for over 2 months now lol.

**Comment by adifaidz** [comment]
I remembered now, the backend site (I think, I may be wrong ) for this app is down https://dalian.toshocat.com/,  https://atarashii.toshocat.com/. And yeah, I think the api for kitsu now has changed (I may also be wrong on this), toshocat is using https://kitsu.io/api/v1/

**Comment by sachaw** [comment]
I'm having the same issue,hopefully it comes back up soon.

**Comment by dmxt** [comment]
I have the same issue. 2 months later now.

**Comment by sachaw** [comment]
+1 Would love to see this worked on

**Comment by matthewdias** [comment]
yep, kitsu api url has changed. api docs: http://docs.kitsu.apiary.io

**Comment by wopian** [comment]
>And yeah, I think the api for kitsu now has changed (I may also be wrong on this), toshocat is using https://kitsu.io/api/v1/

That's actually requesting https://hummingbird.me/api/v1, which was the endpoint before the switch-over to Kitsu
